content/fr/_index.md

```diff
@@ -0,0 +1,90 @@
+---
+title: OpenTelemetry
+description: >-
+  Une télémétrie de qualité, universelle et portable pour une observabilité efficace
+outputs:
+  - HTML
+  - REDIRECTS # Include this `content/fr` ONLY
+developer_note:
+  The blocks/cover shortcode (used below) will use as a background image any
+  image file containing "background" in its name.
+show_banner: true
+---
+
+<div class="d-none"><a rel="me" href="https://fosstodon.org/@opentelemetry"></a></div>
+
+{{< blocks/cover image_anchor="top" height="max" color="primary" >}}
+
+<!-- prettier-ignore -->
+![OpenTelemetry](/img/logos/opentelemetry-horizontal-color.svg)
+{.otel-logo}
+
+<!-- prettier-ignore -->
+{{% param description %}}
+{.display-6}
+
+<div class="l-primary-buttons mt-5">
+
+- [En savoir plus](docs/what-is-opentelemetry/)
+- [Tester la démo](docs/demo/)
+
+</div>
+
+<div class="h3 mt-4">
+<a class="text-secondary" href="docs/getting-started/">Découvrez OpenTelemetry</a> en tant que:
+</div>
+<div class="l-get-started-buttons">
+
+- [Dev](docs/getting-started/dev/)
+- [Ops](docs/getting-started/ops/)
```

Comment:
While dev/ops are understandable, they are englicisms. Should we find proper french words?

Reply:
@dmathieu , would "Dev/Sys Admin" work for you?

Reply:
Technically, `Sys Admin` is still an anglicism. The term should be `Administrateur Système`.
But that term is rather ancient, and really refers to how systems were managed 20 years ago, so I'm not sure it's a good idea to use it.

Reply:
@dmathieu , thanks for your feedback. Will SRE work better?

Reply:
I'd tend to keep `Ops` then.